src/classes/PobControllerExtTest.cls

```diff
@@ -363,6 +363,7 @@ private class PobControllerExtTest {
 	        PageReference nextPage = ce.save();
 	        System.assertEquals(sc.view().getUrl(), nextPage.getUrl(), 'Unexpected page reference');
 			List<Order__c> orders = [SELECT Id, Name FROM Order__c];
+		List<Order__c> orders2 = [SELECT Id, Name FROM Order__a];
```

Comment:
`orders2` does not match naming conventions. Please rename (hint: `myVariable`).